### ISSUE TITLE
ValueError: Procedure called with not enough arguments 

### ISSUE THREAD

**Issue by sweetsugarh** [sbraz/pymediainfo#48]
Hello, I have some problem and don't know why. If I put the mediainfo.dll on the base dir where the script are that works well. But if I try get the .dll manually don't working.

```
media_info = MediaInfo.parse(
	file_mkv,
	library_file ='c:/mediaInfo/MediaInfo.dll'
	)

for track in media_info.tracks:
	if track.track_type == "Video":
		file_library.append(track.writing_library)
```
I get this error:
Python36-32\lib\site-packages\pymediainfo\__init__.py", line 201, in parse
    lib_version = lib.MediaInfo_Option(None, "Info_Version", "")
ValueError: Procedure called with not enough arguments (12 bytes missing) or wrong calling convention

**Comment by sbraz** [comment]
Are you using a 64-bit Python? What about the DLL?
What if you put the library in System32, does it work too?

**Comment by sweetsugarh** [comment]
Im using python 32-bit and the mediainfo dll is 32bits also. If I put the .dll on system32 it not work either. It's only work if i put the .dll on the dir of the script.

**Comment by sbraz** [comment]
Can you attach the DLL you're using here? I'll try to load it with a 32-bit Python 3.6.

**Comment by sweetsugarh** [comment]
Im using their official DLL. attached.

**Comment by sbraz** [comment]
Hmmm it's not showing up.

**Comment by sbraz** [comment]
Can you try replacing `CDLL` with `WinDLL` here?
https://github.com/sbraz/pymediainfo/blob/master/pymediainfo/__init__.py#L139
If that solves the issue, I'll change the code.

**Comment by sweetsugarh** [comment]
I keep giving the same error:

 File "C:\Users\makeinn\AppData\Local\Programs\Python\Python36-32\lib\site-packages\pymediainfo\__init__.py", line 201, in parse
    lib_version = lib.MediaInfo_Option(None, "Info_Version", "")
ValueError: Procedure called with not enough arguments (12 bytes missing) or wrong calling convention

**Comment by sbraz** [comment]
That seems to work for me, I'll check whether this works with a 64-bit Python and release a new version ASAP.

**Comment by sweetsugarh** [comment]
Some idea why that not work for me ?

**Comment by sbraz** [comment]
Did you edit the right file? `C:\Users\makeinn\AppData\Local\Programs\Python\Python36-32\lib\site-packages\pymediainfo\_init_.py` ?

**Comment by sweetsugarh** [comment]
yes

**Comment by sbraz** [comment]
Can you with the `nt_library` branch which I just created?
```pip install https://github.com/sbraz/pymediainfo/archive/nt_library.zip```

**Comment by sweetsugarh** [comment]
Thanks for you work @sbraz . Work fine.

**Comment by sbraz** [comment]
I will release a new version soon then :)

**Comment by sbraz** [comment]
@sweetsugarh I released v2.2.1 which includes this fix.
